cg_openmm/build/cg_build.py

```diff
@@ -495,8 +517,8 @@ def test_force(cgmodel, force, force_type=None):
         for nonbonded_interaction in cgmodel.nonbonded_interaction_list:
             particle_1_positions = cgmodel.positions[nonbonded_interaction[0]]
             particle_2_positions = cgmodel.positions[nonbonded_interaction[1]]
-            sigma = cgmodel.get_particle_sigma(nonbonded_interaction[0])
-            epsilon = cgmodel.get_particle_epsilon(nonbonded_interaction[0])
```

Comment:
Is this reverting some of the changes recently added?  get_particle_sigma is the new version of the function, not get_sigma